base/scripts/lib/install_phantomjs.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 apt-get -y install libfreetype6 libfreetype6-dev fontconfig
 ARCH=`uname -m`
-PHANTOMJS_VERSION=1.9.8
+PHANTOMJS_VERSION=2.1.1
```

Comment:
I've intentionally avoided this update in the past because it's been such a breaking change, historically speaking.  Anything you can do to make me feel better about it? (Keeping that this image is a bit legacy — originally intended as a plug-in replacement for `meteorhacks/meteor` — in some regards.)

Reply:
The reason I updated is because this version of PhantomJS would be downloaded on every creation of a new Docker instance for our apps, so I thought I might as well include it to begin with.

If this image is a bit legacy, could you point me to a non-legacy version (other than my own fork ;))?

Reply:
Looks nice, but I'm not sure this can be easily used together with mup?